systemtests/tests/restore-test/testrunner-full-restore

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+#
+# Do a full restore from an archive job.
+#
+
+TestName="$(basename "$(pwd)")"
+export TestName
+
+JobName=backup-bareos-fd
+
+#shellcheck source=../environment.in
+. ./environment
+
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+
+start_test
+
+cat <<END_OF_DATA >"$tmp/bconcmds"
+@$out /dev/null
+messages
+@$out $tmp/setup.out
+setdebug level=100 storage=File
+status director
+status client
+status storage=File
+wait
+messages
+@$out $tmp/jobs.out
+list jobs
+@$out $tmp/backup-full.out
+restore client=bareos-fd fileset=SelfTest where=$tmp/bareos-restores select all done yes
+wait
+messages
+quit
+END_OF_DATA
+
+run_bconsole
+check_for_zombie_jobs storage=File
+
+# check the most current jobs list form $tmp/jobs.out and check that it is
+# indeed a backup
+if ! grep "1 | backup-bareos-fd" "$tmp/jobs.out" | tail -n 1 | grep "| B" > /dev/null; then
+    echo "Job 1 has to always be a backup job at the start of a test."
+    exit 5
+fi
+
+expect_grep "Start Restore Job" \
+	    "$tmp/should-work.out" \
+	    "Required restore job was not started."
```

Comment:
```suggestion
expect_grep "Start Restore Job" \
	    "$tmp/backup-full.out" \
	    "Required restore job was not started."
```
Beware you didn't defined should-work.out in this test, so you're checking something in the other test.
So the question is beside the simple fix, how can we protect others from doing the same mistake in next maintenance cycle?

I believe we maybe need a function that cleanup the tmp/*.out after each tests, what's your thought?

Reply:
It is up to you to come with a solution where we can keep all logs, because they are unique ;-) I completely agree with the keeping.
btw at the end of all tests you still need to clean up everything (see test-clean up on other case) so the test can be rerun again.

Reply:
I added my "fix" to the last batch of commits.  What do you think about that idea @pstorz ?  Is that something we should do going forward ?

I think currently a lot of tests are just using log1, log2 etc.

Reply:
we agreed to do what we are doing like it is for the moment.